dataactbroker/handlers/submission_handler.py

```diff
@@ -96,11 +96,8 @@ def get_submission_status(submission, jobs):
     skip_count = 0
 
     for job in jobs:
-        if job.job_type.name not in ["external_validation", None]:
```

Comment:
Was there a reason we were checking for the `None`? Do we want to keep checking for it?